doc/api/sqlite.md

```diff
@@ -326,7 +326,7 @@ added: v22.5.0
 
 * `namedParameters` {Object} An optional object used to bind named parameters.
   The keys of this object are used to configure the mapping.
-* `...anonymousParameters` {null|number|bigint|string|Buffer|Uint8Array} Zero or
+* `...anonymousParameters` {null|number|bigint|string|Buffer|TypedArray|DataView} Zero or
```

Comment:
Could you add a `changes` list in the YAML comment to track on what versions support has been backported please?